R/model.R

```diff
@@ -61,6 +61,31 @@ epirange <- function(from, to) {
   structure(list(from = from, to = to), class = "EpiRange")
 }
 
+#' @export
+print.EpiRange <- function(x, ...) {
+  stopifnot(inherits(x, "EpiRange"))
+
+  if (nchar(x$from) == 8) {
+    date_type <- "Days" # nolint: object_usage_linter
+    x$from <- as.Date(as.character(x$from), "%Y%m%d")
+    x$to <- as.Date(as.character(x$to), "%Y%m%d")
+  } else if (nchar(x$from) == 6) {
+    date_type <- "Epiweeks" # nolint: object_usage_linter
+    x$from <- format(
+      as.Date(as.character(x$from), "%Y%U"),
```

Comment:
issue: `%U` != US epiweek convention.  Both use Sunday as start of the week, but in epiweek convention, weeks are numbered 1 to 53 and assigned to a year based on what year their Wednesday falls on.

issue: this always outputs week 0 for some reason.  Maybe `as.Date` failing due to a wday not being provided?
```
epirange(200101, 200230)

── <EpiRange> object: ────────────────────────────────────────────────────────────────────────
Epiweeks from 2001w00 to 2002w00
```

Reply:
Good catch! Avoid date conversion for simplicity.

Reply:
Sounds good; that's also more general.  Will be good if we add non-US-epiweeks, though the current wording ("Epiweeks") will need an update if we do.  (I don't think we have non-US-epiweeks anywhere yet (besides the `collection_weeks` which are actually specified as dates).)